src/EditorFeatures/VisualBasicTest/Diagnostics/ImplementInterface/ImplementInterfaceTests.vb

```diff
@@ -572,6 +572,14 @@ NewLines("Imports System.ComponentModel \n Class C \n Implements [|INotifyProper
 NewLines("Imports System.ComponentModel \n Class C \n Implements INotifyPropertyChanged \n Public Event PropertyChanged As PropertyChangedEventHandler Implements INotifyPropertyChanged.PropertyChanged \n End Class"))
         End Function
 
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsImplementInterface)>
+        Public Async Function TestImplementThroughMemberEvent() As Task
+            Await TestAsync(
+NewLines("Imports System.ComponentModel \n Class Slave \n Implements INotifyPropertyChanged \n Public Event PropertyChanged As PropertyChangedEventHandler Implements INotifyPropertyChanged.PropertyChanged \n End Class \n Class Master \n Implements [|INotifyPropertyChanged|] \n Private slave As Slave \n End Class"),
+NewLines("Imports System.ComponentModel \n Class Slave \n Implements INotifyPropertyChanged \n Public Event PropertyChanged As PropertyChangedEventHandler Implements INotifyPropertyChanged.PropertyChanged \n End Class \n Class Master \n Implements INotifyPropertyChanged \n Private slave As Slave \n Public Custom Event PropertyChanged As PropertyChangedEventHandler Implements INotifyPropertyChanged.PropertyChanged \n AddHandler(value As PropertyChangedEventHandler) \n AddHandler DirectCast(slave, INotifyPropertyChanged).PropertyChanged, value \n End AddHandler \n RemoveHandler(value As PropertyChangedEventHandler) \n RemoveHandler DirectCast(slave, INotifyPropertyChanged).PropertyChanged, value \n End RemoveHandler \n RaiseEvent(sender As Object, e As PropertyChangedEventArgs) \n End RaiseEvent \n End Event \n End Class"),
+            1)
```

Comment:
wrap lines longer than github.

Note: NewLines is not necessary for VB tests anymore.  We needed it back when VB didn't have multi-line strings.  But those days are over.